core/src/main/java/org/apache/accumulo/core/trace/TraceUtil.java

```diff
@@ -131,48 +176,6 @@ public static void setException(Span span, Throwable e, boolean rethrown) {
     }
   }
 
-  /**
-   * Obtain {@link org.apache.accumulo.core.clientImpl.thrift.TInfo} for the current context. This
-   * is used to send the current trace information to a remote process
-   */
-  public static TInfo traceInfo() {
-    TInfo tinfo = new TInfo();
-    W3CTraceContextPropagator.getInstance().inject(Context.current(), tinfo, TInfo::putToHeaders);
-    return tinfo;
-  }
-
-  /**
-   * Returns a newly created Context from the TInfo object sent by a remote process. The Context can
-   * then be used in this process to continue the tracing. The Context is used like:
-   *
-   * <pre>
-   * Context remoteCtx = getContext(tinfo);
-   * Span span = tracer.spanBuilder(name).setParent(remoteCtx).startSpan()
-   * </pre>
-   *
-   * @param tinfo tracing information serialized over Thrift
-   */
-  private static Context getContext(TInfo tinfo) {
-    return W3CTraceContextPropagator.getInstance().extract(Context.current(), tinfo,
-        new TextMapGetter<TInfo>() {
-          @Override
-          public Iterable<String> keys(TInfo carrier) {
-            if (carrier.getHeaders() == null) {
-              return null;
-            }
-            return carrier.getHeaders().keySet();
-          }
-
-          @Override
-          public String get(TInfo carrier, String key) {
-            if (carrier.getHeaders() == null) {
-              return null;
-            }
-            return carrier.getHeaders().get(key);
-          }
-        });
-  }
-
```

Comment:
These two old methods were basically serialize/deserialize. We can replace them with new serialize/deserialize methods, even though the type is not TInfo, but some other serialized form (String).